preprocessing/nextclade/src/loculus_preprocessing/backend.py

```diff
@@ -173,6 +178,28 @@ def submit_processed_sequences(
     logger.info("Processed data submitted successfully")
 
 
+def request_upload(group_id: int, number_of_files: int, config: Config) -> Sequence[FileUploadInfo]:
+    # we need to parse the backend URL, to extract the API path without the organism component
+    parsed = urlparse(config.backend_host)
+    base_url = f"{parsed.scheme}://{parsed.netloc}"
+    url = base_url + "/files/request-upload"
+    params = {"groupId": group_id, "numberFiles": number_of_files}
+    headers = {"Authorization": "Bearer " + get_jwt(config)}
+    response = requests.post(url, headers=headers, params=params, timeout=10)
+    if not response.ok:
+        msg = f"Upload request failed: {response.status_code}, {response.text}"
+        raise RuntimeError(msg)
+    return [FileUploadInfo(**item) for item in response.json()]
+
+
+def upload_embl_file_to_presigned_url(content: str, url: str) -> None:
+    headers = {"Content-Type": "chemical/x-embl"}
```

Comment:
I think the correct type is maybe "chemical/x-embl-dl-nucleotide" based on https://en.wikipedia.org/wiki/Chemical_file_format

```suggestion
    headers = {"Content-Type": "chemical/x-embl-dl-nucleotide"}
```